packager/version/generate_version_string.py

```diff
@@ -33,7 +33,7 @@ def check_output_implementation(*popenargs, **kwargs):
   try:
     version_tag = subprocess.check_output(
         ['git', 'tag', '--points-at', 'HEAD'],
-        stderr=subprocess.STDOUT).rstrip()
+        stderr=subprocess.STDOUT, shell=True).rstrip()
```

Comment:
This is failing on Linux and MAC. I think you need to change ['git', 'tag', '--points-at', 'HEAD'] to ['git tag --points-at HEAD']. The same below.

Reply:
The new code succeeds in Linux / MAC, but fails on Windows, can you try if removing "[]" works on both Linux and Windows?

If not, we'll have to make shell=True conditional on Windows.